src/constants/indicators.js

```diff
@@ -179,59 +179,89 @@ export default {
+  },
+  educationalAttainmentTertiary: {
+    name: 'Educational attainment - Tertiary',
+    shortDescription: 'TODO',
+    longDescription: 'TODO',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Education attainment - Completed Bachelor Degree or Higher, % of population',
+  },
   dispersion: {
     name: 'Local government dispersion',
-    shortDescription: 'Local government dispersion',
-    longDescription: 'Local government dispersion',
+    shortDescription: 'LGAs per 100,000 persons',
+    longDescription: 'Number of Local Government Areas per 100 000 persons',
     dataType: DATA_TYPES.NUMBER,
     aggregationMethod: AGGREGATION_METHODS.MEDIAN,
     format: '0[.]0%',
     contextual: false,
     source: 'Government Dispersion, Number of Local Government Areas per 100 000 persons',
   },
   dwellingHouse: {
-    name: 'House',
-    shortDescription: 'TODO',
-    longDescription: 'TODO',
+    name: 'Dwelling type ratio: house',
+    shortDescription: 'House',
+    longDescription: 'Dwelling type ratio: house',
     dataType: DATA_TYPES.NUMBER,
     aggregationMethod: AGGREGATION_METHODS.MEDIAN,
     format: '0[.]0%',
     contextual: false,
     source: 'Dwelling type ratio, Separate house, Percentage of Dwellings',
   },
   dwellingSemi: {
-    name: 'Semi-House',
-    shortDescription: 'TODO',
-    longDescription: 'TODO',
+    name: 'Dwelling type ratio: semi-detatched',
+    shortDescription: 'Semi-detatched',
+    longDescription: 'Dwelling type ratio: semi-detatched',
     dataType: DATA_TYPES.NUMBER,
     aggregationMethod: AGGREGATION_METHODS.MEDIAN,
     format: '0[.]0%',
     contextual: false,
     source: 'Dwelling type ratio, Semi-detached, Percentage of Dwellings',
   },
   dwellingApartment: {
-    name: 'Apartment',
-    shortDescription: 'TODO',
-    longDescription: 'TODO',
+    name: 'Dwelling type ratio: flat or apartment',
+    shortDescription: 'Flat / apartment',
+    longDescription: 'Dwelling type ratio: flat or apartment',
     dataType: DATA_TYPES.NUMBER,
     aggregationMethod: AGGREGATION_METHODS.MEDIAN,
     format: '0[.]0%',
     contextual: false,
     source: 'Dwelling type ratio, Flat or Apartment, Percentage of Dwellings',
   },
   dwellingOther: {
-    name: 'Other',
-    shortDescription: 'TODO',
-    longDescription: 'TODO',
+    name: 'Dwelling type ratio: other',
+    shortDescription: 'Other',
+    longDescription: 'Dwelling type ratio: other',
     dataType: DATA_TYPES.NUMBER,
     aggregationMethod: AGGREGATION_METHODS.MEDIAN,
     format: '0[.]0%',
     contextual: false,
     source: 'Dwelling type ratio, Other, Percentage of Dwellings',
   },
+  dwellingPriceToIncomeRatio: {
+    name: 'Dwelling price to household income ratio',
+    shortDescription: 'Dwelling price to household income ratio',
+    longDescription: 'Dwelling price to household income ratio',
+    dataType: DATA_TYPES.NUMBER,
```

Comment:
If the indicators are all done now and `dataType` is always `DATA_TYPES.NUMBER` then this logic can be removed (now or in a new Jira task).

Reply:
In this task is fine - but should that also include removing references to this elsewhere in the code ?

Reply:
Yep. For memory I was checking it in the conversion file

Reply:
Also, what about AGGREGATION_METHODS? Is that still needed?

Reply:
It's gone in my branch that's ready for you to review.